CMakeLists.txt

```diff
@@ -1,13 +1,18 @@
 if(CCPP)
 
+  if(DEBUG)
```

Comment:
This change (everything related to `_ccpp_debug_arg`) was merged into develop a few weeks back, see https://github.com/NOAA-EMC/fv3atm/blob/develop/CMakeLists.txt. Why is it showing up here again?

Reply:
I don't know why the files show up here again, but when comparing directly to origin/develop using git, there is only a change to the CMakeLists.txt file that changes the ccppphysics link to PUBLIC.